core/src/components/header/test/divider/header.e2e.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('header: divider'), () => {
+    test('should not have visual regressions with divider header', async ({ page }) => {
+      await page.goto(`/src/components/header/test/divider`, config);
+
+      await page.setIonViewport();
```

Comment:
Capturing full page screenshots will slow CI performance over time. Since the screenshot is only for the header, you could use `.setContent` of the header with a bit of padding to include the divider/shadow.

Reply:
Thanks, always forget about that 😂